docs/wrapping-react/example.md

```diff
@@ -37,10 +37,11 @@ Here we will define the `tag` and the `vars` that we will need to use the compon
 For this tutorial we will define `EventHandler` props `on_nodes_change` and `on_connect`, but you can find all the events that the component triggers in the [reactflow docs](https://reactflow.dev/docs/api/react-flow-props/#onnodeschange).
 
 ```python
-from reflex import Component, EventHandler, Var
+import reflex as rx
+from reflex import EventHandler, Var
```

Comment:
we can also do `rx.EventHandler` and `rx.Var`

Reply:
changed this on all pages